pandas/tests/io/formats/style/test_to_latex.py

```diff
@@ -239,15 +246,32 @@ def test_multiindex_row(df):
         \\end{tabular}
         """
     )
-    assert expected == s.to_latex(sparse_index=False)
+    result = styler.to_latex(sparse_index=False)
+    assert expected == result
+
+
+def test_multirow_naive(df_ext):
+    ridx = MultiIndex.from_tuples([("X", "x"), ("X", "x"), ("Y", "y")])
```

Comment:
Maybe better to avoid a duplicate in the tuples (like ``("X", "x")``).

This would do:

``` python
[("X", "x"), ("X", "z"), ("Y", "y")]
```